Automattic/wp-calypso
# Redesign ready to blaze Posts

Related to https://jira.tumblr.net/browse/SELFSERVE-408

## Proposed Changes
Redesign includes
- Revamped table for Ready to blaze posts,
- Use a search bar (previously added for campaigns)
- Revamped tab panel

## Testing Instructions



- Run Calypso locally,
- Login as A8C user,
- Check out Tools > Advertising > Posts ([local link](http://calypso.localhost:3000/advertising/timurads.wordpress.com/posts))
- Try out dropdown,

<img width="1100" alt="image" src="https://user-images.githubusercontent.com/115007291/233160240-e3feb673-1595-4b17-90f3-b45cc3bf4697.png">

- Switch to Campaigns tab and check whether they work as expected
<img width="1102" alt="image" src="https://user-images.githubusercontent.com/115007291/233161217-4a639dec-d0de-4a55-8d28-4bef5ec591d6.png">

## Pre-merge Checklist
